File: level36/homework/codwars4.py
#https://www.codewars.com/kata/57a5b0dfcf1fa526bb000118/train/python
def distinct(seq):
    seen = set()
    result = []
    for item in seq:
        if item not in seen:
            seen.add(item)
            result.append(item)
    return result

File: level56/homework/codwars4.js
//https://www.codewars.com/kata/56b1f01c247c01db92000076/train/javascript
function doubleChar(str) {
	var word = '';
  for (var i = 0; i < str.length; i++){
  	word = word + str[i] + str[i];
  };
  return word;
};